src/parameter_bridge.cpp

```diff
@@ -85,9 +95,93 @@ int main(int argc, char * argv[])
     }
   } else {
     fprintf(
-      stderr, "The parameter '%s' either doesn't exist or isn't an array\n", parameter_name);
+      stderr, "The parameter '%s' either doesn't exist or isn't an array\n", topics_parameter_name);
   }
 
+  // ROS 1 Services in ROS 2
+  XmlRpc::XmlRpcValue services_1_to_2;
+  if (
+    ros1_node.getParam(services_1_to_2_parameter_name, services_1_to_2) &&
+    services_1_to_2.getType() == XmlRpc::XmlRpcValue::TypeArray)
+  {
+    for (size_t i = 0; i < static_cast<size_t>(services_1_to_2.size()); ++i) {
+      std::string service_name = static_cast<std::string>(services_1_to_2[i]["service"]);
+      std::string package_name = static_cast<std::string>(services_1_to_2[i]["package"]);
+      std::string type_name = static_cast<std::string>(services_1_to_2[i]["type"]);
+      printf(
+        "Trying to create bridge for ROS1 service '%s' "
```

Comment:
The later called function `service_bridge_1_to_2` creates a ROS 2 client and a ROS 1 server. Shouldn't the message then say "bridge got ROS 2 service"?

Same inversion below.